cmake/3rdPartyPackages.cmake

```diff
@@ -679,6 +694,16 @@ macro(ly_set_package_download_location package_name download_location)
     set_property(GLOBAL PROPERTY LY_PACKAGE_DOWNLOAD_LOCATION_${package_name} ${download_location})
 endmacro()
 
+# ly_set_package_download_cache_location - OPTIONAL.
+# Similar to ly_set_package_download_location, this set the download 
+# cache location for a particular package.
+# note that package_name is expected to be the actual package name, not 
+# the find_package(...)  name!
```

Comment:
is there an example of the difference between these 2 names? what to look out for?

Reply:
the find_package name is the name of the FindXXXXX.cmake file to run, so for example, "ZLIB" to run FindZLIB.cmake.
the package name is literally the file name of the package, like `zlib-multiplatform-1.2.3-rev1` (without the .tar.xz)